### PR TITLE
Order show page

### LINKED ISSUE #46: User Story 29, User views an Order Show Page
```
As a registered user
When I visit my Profile Orders page
And I click on a link for order's show page
My URL route is now something like "/profile/orders/15"
I see all information about the order, including the following information:
- the ID of the order
- the date the order was made
- the date the order was last updated
- the current status of the order
- each item I ordered, including name, description, thumbnail, quantity, price and subtotal
- the total quantity of items in the whole order
- the grand total of all items for that order
```

DO THESE STORIES FIRST:
- User Story 28, User Profile displays Orders

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #49

child of #82

depends on #47

### PR DESCRIPTION
**Pull request by renecasco**
- [] Check this if the PR has been approved by the team to be a quick patch and the rest of this form will not be filled out

# Test Coverage Percentage (Model/Feature):
- rspec - 100%
- rspec spec/models - 98.41%

# What functionality does this accomplish?
User Order show page 

# What did you struggle on to complete?

# Implements/Fixes:
* description
closes #46 

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

# Please include an emoji of how you feel about this branch:
👍

### CODE REVIEW

File: spec/models/item_spec.rb [hunk 1]
```diff
@@ -42,13 +42,13 @@
         expect(item.average_fulfilled_time).to include("3 days 16:00")
       end
 
-      it 'should calculate average fulfillment time for an item' do
+      it 'should calculate average fulfillment time for an item' do #this test has same name as previous one
```
Comment (milevy1) [inline spec/models/item_spec.rb]: We should change the it 'message' to say something different

Comment (milevy1) [reply spec/models/item_spec.rb]: I went ahead and changed it.

File: spec/features/reg_users/order_show_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        @item_1 = create(:item)
+        @item_2 = create(:item)
+
+        @order_item_1= create(:fulfilled_order_item, item: @item_1, order: @order_1)
+        @order_item_2= create(:fulfilled_order_item, item: @item_2, order: @order_1, quantity: 3)
+
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+
+        visit profile_orders_path
+        click_on "#{@order_1.id}"
```
Comment (blake-enyart) [inline spec/features/reg_users/order_show_page_spec.rb]: Just for reference, you should just be able to put in only @order_t.id.to_s if that is easier to understand in the future
